### DISCUSSION TITLE
argo-workflows/cli/argo_version/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#8883] [General]
# argo-workflows/cli/argo_version/

https://argoproj.github.io/argo-workflows/cli/argo_version/

**Comment by ezk84** [comment]
Is is possible to determine the server version using this command? It seems to only return the cli's version.

**Reply by terrytangyuan** [reply]
If you specify `ARGO_SERVER`, it should also print out the server version.

**Reply by ezk84** [reply]
I am running argo behind an ssl-enforcing ingress in a cluster, and I am able to access the UI just fine from outside the cluster on https://ingresshostname

If I on the other hand set ARGO_SERVER=ingresshostname:443 and call `argo --verbose version` the I get an error:

```
time="2022-08-26T13:15:45.611Z" level=debug msg="CLI version" version="{v3.4.0-rc1 2022-08-10T06:55:47Z e5c587b5a9274acb6c7c8884e23d1608b37bb3e5 v3.4.0-rc1 clean go1.18.5 gc windows/amd64}"
argo: v3.4.0-rc1
  BuildDate: 2022-08-10T06:55:47Z
  GitCommit: e5c587b5a9274acb6c7c8884e23d1608b37bb3e5
  GitTreeState: clean
  GitTag: v3.4.0-rc1
  GoVersion: go1.18.5
  Compiler: gc
  Platform: windows/amd64
time="2022-08-26T13:15:45.612Z" level=debug msg="Client options" opts="(argoServerOpts=(url=ingresshostname:443,path=,secure=true,insecureSkipVerify=false,http=false),instanceID=)"
I0826 13:15:45.613557  523116 loader.go:372] Config loaded from file:  C:\Users\Eze\.kube\config
I0826 13:15:46.684748  523116 round_trippers.go:553] GET https://cluster's ip address 200 OK in 1037 milliseconds
time="2022-08-26T13:15:47.933Z" level=fatal msg="rpc error: code = Unknown desc = transport: received unexpected content-type \"text/html; charset=utf-8\""
```

What am I doing wrong here?

**Reply by sstaley-hioscar** [reply]
For future reference, try export ARGO_HTTP1=true for `received unexpected content-type \"text/html;`
